scripts/upgrade_pysqlite.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Upgrade the version of pysqlite/sqlite to avoid crashes during testing.
+# Ideally, this code would just install a pre-built wheel - change it to this
+# once edX has its own artifact server.
+pip uninstall -y pysqlite
```

Comment:
Is there a good way to determine quickly if we've already run this, so we don't need to do this installation on every test run in the same virtualenv?

Reply:
I don't know of one, since pysqlite has the same version post-installation as the one in testing.txt:
```
pysqlite==2.8.3
```
The script *could* poke around in both the machine venv and the tox venv. But that seems error-prone. Or the script could leave behind a file of some sort as a flag, perhaps. Do you know of a way?

Reply:
Is it installing a different version of sqlite than the default?  If so, could check the expected version via:

```python
import sqlite3
sqlite3.sqlite_version
```

Reply:
@jmbowman I found a way to check - and I've added the check to the installation script. Ready for re-review.